modules/ROOT/pages/limitations-self.adoc

```diff
@@ -5,6 +5,9 @@ endif::[]
 
 Review the following requirements and limitations before installing, configuring, and managing Anypoint Runtime Fabric. 
 
+[[license-compliance]]
+include::partial$include-deploy-resource-allocation.adoc[tags=licenseCompliance;!instructionsMonitor]
```

Comment:
Is the ! intentional in the tags values? I've just never seen it  before, so if it's fine, ignore me.